uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/RegressionTest.java

```diff
@@ -183,7 +183,7 @@ public void issue_1076() {
 		new WaitUntil(new ShellIsAvailable("New Server Runtime Environment"));
 		new DefaultShell("New Server Runtime Environment").setFocus();
 		AbstractWait.sleep(TimePeriod.SHORT);
-		new DefaultTreeItem("JBoss Fuse", "JBoss Fuse 6.1").select();
+		new DefaultTreeItem("Red Hat Fuse", "Red Hat Fuse 6.1").select();
```

Comment:
need to be kept as-is for 6.x https://issues.jboss.org/browse/FUSETOOLS-2843?focusedCommentId=13545002&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-13545002
(but perhaps only the second-level as the first level might be shared)